python/smashbox/utilities/__init__.py

```diff
@@ -279,6 +279,16 @@ def run_ocsync(local_folder, remote_folder="", n=None, user_num=None):
 def webdav_propfind_ls(path):
     runcmd('curl -s -k %s -XPROPFIND %s | xmllint --format -'%(config.get('curl_opts',''),oc_webdav_url(remote_folder=path)))
 
+def expect_webdav_does_not_exist(path):
+    exitcode,stdout,stderr = runcmd('curl -s -k %s -XPROPFIND %s | xmllint --format - | grep NotFound | wc -l'%(config.get('curl_opts',''),oc_webdav_url(remote_folder=path)))
+    exists = stdout.rstrip() == "1"
+    error_check(exists, "Remote path does not %s exist but should" % path)
+
+def expect_webdav_exist(path):
+    exitcode,stdout,stderr = runcmd('curl -s -k %s -XPROPFIND %s | xmllint --format - | grep NotFound | wc -l'%(config.get('curl_opts',''),oc_webdav_url(remote_folder=path)))
```

Comment:
hard coded user_num=None here because it is not tunnel... quite unfortunate because this means the method can not be used in multi user tests.
But okay for now. will take care of it in my "rebase" plan.